Improve --quiet flag wording in CLI help

<!-- Please answer the following. Issues that do not will be closed. -->

> Clearly describe the bug

When using the `--quiet` flag in the stylelint cli should I expect the errors to be changed to warnings and the cli to exit with exit code 0 ? Currently the quiet option doesn't seem to do much at all.

> Which rule, if any, is the bug related to?

Related to CLI tool. using `-q` or `--quiet`

> What CSS is needed to reproduce the bug?

```css
.emtpy-block {

}
```

> What stylelint configuration is needed to reproduce the bug?

default config or no config

> Which version of stylelint are you using?

10.1.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

running inside as npm script
 ```
"scripts": {
    "test": "stylelint test.css --quiet"
}
```

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

Errors to be converted to warnings and the cli to exit quietly if it is able to lint correctly, but exit with code 1

> What actually happened (e.g. what warnings or errors did you get)?

`npm ERR! Test failed.  See above for more details.`

When I pass `--print-config` I can see that quiet is set to true.

--- a/lib/cli.js
+++ b/lib/cli.js
@@ -264,7 +264,7 @@ const meowOptions /*: meowOptionsType*/ = {
 
       --quiet, -q
 
-        Only register warnings for rules with an "error"-level severity (ignore
+        Only register violations for rules with an "error"-level severity (ignore
         "warning"-level).
 
       --color

